### DISCUSSION TITLE
syzygy wdl probing

### DISCUSSION THREAD

**Discussion by robertnurnberg** [niklasf/python-chess#1093] [Q&A] [answered]
It seems to me that `get_wdl()` can return `2` in a 7men position even if only 6men EGTBs are loaded, if there is a winning capture into 6men. An example is `8/8/3k4/R2b4/Q7/4p3/1K6/6n1 w - -`.

To me it seems that the same does not happen for an 8men position that captures into 7men, if full 7men EGTBs are loaded.

Is that correct? Also, is that a design decision or a quirk of the current implementation? @niklasf

**Comment by niklasf** [comment, marked as answer]
That's a quirk of the implementation. Fixed via https://github.com/niklasf/python-chess/commit/a41c3c88022fd97ab0eaf7b032a3158b69628b66.

**Reply by robertnurnberg** [reply]
Thanks. I understand that the commit means that now also 8men positions are being checked for having winning captures into 7men EGTBs (if they are loaded). Correct?

**Reply by niklasf** [reply]
Correct.
